tests/cli/test_suite.py

```diff
@@ -2356,18 +2354,20 @@ def test_suite_list_with_one_suite(
     mock_emit, caplog, monkeypatch, empty_data_context_stats_enabled
 ):
     context = empty_data_context_stats_enabled
-    monkeypatch.chdir(os.path.dirname(context.root_directory))
+    monkeypatch.chdir(os.path.dirname(context.root_directory))  # noqa: PTH120
 
     project_dir: str = context.root_directory
 
-    config_file_path: str = os.path.join(project_dir, "great_expectations.yml")
-    assert os.path.exists(config_file_path)
+    config_file_path: str = os.path.join(  # noqa: PTH118
+        project_dir, "great_expectations.yml"
+    )
+    assert os.path.exists(config_file_path)  # noqa: PTH110
 
     expectation_suite_dir_name: str = "a_dir"
     expectation_suite_name: str = "test_suite_name"
 
     # noinspection PyUnusedLocal
-    suite: ExpectationSuite = context.add_expectation_suite(
```

Comment:
Do we not want to save the suite returned from add_expectation_suite in these 2 places?